Bug in Item3D::IsAboveBelow

gustav21 opened the pull request:
When calculating `ol2` there must be `ol1` instead of `ow2`

gustav21 committed:
Bug in Item3D::IsAboveBelow

When calculating 'ol2' there must be 'ol1' instead of 'ow2'

JamesBremner reviewed (changes requested):
This looks correct.  There is a unit test of IsAboveBelow().  Did the test miss this?  Can you add to the unit test so that the bug is caught?

gustav21 committed:
Changes in AboveBelow test

These changes allows IsAboveBelow bug to occur.

JamesBremner reviewed (changes requested):
Thank you.

This looks like you have modified the existing unit test.  Please do not do that!  If you add a new test specifically for the bug you found, then everything will be clear and any bugs the existing test was designed to detect will still be tested.

gustav21 commented:
Ok, sorry, I'll try to add new test for the bug.

gustav21 committed:
Add AboveBelowBug test

1) Made a roollback commit db2c534.
2) Add AboveBelowBug test which tests the fix in commit 1133deb

JamesBremner reviewed (approved):
Thank you
